extensions/wikia/TemplateClassification/TemplateClassification.hooks.php

```diff
@@ -98,12 +106,20 @@ public function onEditPageShowEditFormFields( \EditPage $editPage, \OutputPage $
 		global $wgCityId;
 
 		$context = \RequestContext::getMain();
+		$title = $context->getTitle();
 
-		if ( ( new Permissions() )->shouldDisplayEntryPoint( $context->getUser(), $context->getTitle() ) ) {
+		if ( ( new Permissions() )->shouldDisplayEntryPoint( $context->getUser(), $title ) ) {
 			$types = $this->getTemplateTypeForEdit( $editPage->getTitle(), $wgCityId );
 
-			$out->addHTML( \Html::hidden( 'templateClassificationTypeCurrent', $types['current'] ) );
-			$out->addHTML( \Html::hidden( 'templateClassificationTypeNew', $types['new'] ) );
+			$out->addHTML( \Html::hidden( 'templateClassificationTypeCurrent', $types['current'],
+				['autocomplete' => 'off'] ) );
+			$out->addHTML( \Html::hidden( 'templateClassificationTypeNew', $types['new'],
+				['autocomplete' => 'off'] ) );
+
+			// add additional class to body for new templates in order to hide editor while TCS modal is visible
```

Comment:
Hmm, do we want to do it only for new templates? How we cover existing template with no set type?